onnxscript/test/eval_test.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import numpy as np
+
+from onnxscript import evaluator, graph, script
+from onnxscript.onnx_opset import opset17 as op
+from onnxscript.onnx_types import FLOAT
+
+
+class EvaluatorTest(unittest.TestCase):
+    def test_mixed_evaluator(self):
+        @script()
+        def seq_map(x: FLOAT["N"]):  # noqa: F821
+            seq1 = op.SequenceConstruct(x, x + 1, x + 2)
+
+            @graph()
+            def square(y: FLOAT["N"]) -> FLOAT["N"]:  # noqa: F821
+                return op.Mul(y, y)
+
+            seq2 = op.SequenceMap(seq1, body=square)
+            return seq2
+
+        x = np.array([0.0, 1.0], dtype=np.float32)
+        output = seq_map[evaluator.ort_mixed_evaluator](x)
```

Comment:
It would be better to add another unit test to check a dummy evaluator even if this one is calling ORT underneath.

Reply:
I don't understand. Happy to add a test, but what do you want to test and how?